bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4545,4 +4545,19 @@
         </sql>
     </changeSet>
 
+    <changeSet id="Create-practitioner_role-20220721" author="Abinaya, Siva, Arjun">
```

Comment:
personally would prefer the attribute name being "practitioner_type" .. instead of "practitioner_role", so that there is no ambiguity with roles (privilege) while documenting and discussing

Reply:
renamed  practitioner_role to practitioner_type